code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -26,6 +26,7 @@
 
 /datum/antagonist/ashwalker/on_gain()
 	RegisterSignal(owner.current, COMSIG_MOB_EXAMINATE, .proc/on_examinate)
+	owner.current.remove_language(/datum/language/common) //Ash walkers do not know Intergalactic Common
 
```

Comment:
This should not be part of your PR.

Reply:
Ah fuck, forgot to switch bases before making a commit for something else. Whoops.